test/test_numpy_interop.py

```diff
@@ -153,6 +153,31 @@ def test_to_numpy_bool(self, device) -> None:
         self.assertEqual(y.dtype, np.bool_)
         self.assertEqual(x[0], y[0])
 
+    def test_to_numpy_force_argument(self, device) -> None:
+        for force in [False, True]:
```

Comment:
Style nit:

```
for force, requires_grad, sparse, conj in itertools.product(*(((False, True),) * 4)):
```

will save you so much indenting

Reply:
That said, I think it might make more sense to just create smaller, separate tests for these properties: requires_grad, sparse, and conj.

Reply:
I wasn't really sure how to divide them into smaller tests. My goal was to test every possible case out of 2^4 options. Do you have an idea how to separate them?